sdk/core/azure-core-management/src/main/java/com/azure/core/management/AzureEnvironment.java

```diff
@@ -81,7 +103,7 @@ public AzureEnvironment(Map<String, String> endpoints) {
      * Provides the settings for authentication with Azure US Government.
      */
     public static final AzureEnvironment AZURE_US_GOVERNMENT = new AzureEnvironment(new HashMap<String, String>() {{
-            put("portalUrl", "https://manage.windowsazure.us");
+            put("portalUrl", "https://portal.azure.us");
```

Comment:
All these hardcoded values for each cloud can be taken from the ARM metadata endpoint (instead of hardcoding).  may be in a future PR, no need to address it art of this one :)
https://management.azure.com/metadata/endpoints?api-version=2019-10-01

Reply:
Note that it should be from respective cloud's ARM endpoint , assuming connectivity to the ARM endpoint

Reply:
These are more for convention. And it does save one request, as pre-defined values.

Also current a big issue is that current metadata does not provide Microsoft Graph endpoint, so that if customer uses metadata, they had to provide that endpoint manually.